src/index.js

```diff
@@ -0,0 +1,139 @@
+import './style.css';
+import Fetch from './display.js';
+import { resetData, showPopup } from './modules/views/meal-show.js';
+import {
+  closeBtn, popupBox,
+} from './modules/views/meal-dom-elements.js';
+import { form, feedbackBox } from './modules/views/comment-dom-elements.js';
+import addComment from './modules/comments/add.js';
+import comments from './modules/comments/index.js';
+import setComment from './modules/views/comment-show.js';
+import { setText, showOnSuccess } from './modules/global-value.js';
+
+// declare HTML element to hold the meals display
+const mealsRow = document.querySelector('.meals-row');
+const mealsList = new Fetch();
+
+// get all categories to add event listeners
+const categories = document.querySelectorAll('.category');
+
+const showLikes = async (id) => {
+  let likes = 0;
+  const meals = await mealsList.getLikes();
+  const currentItem = meals.find((meal) => meal.item_id === id);
+  if (currentItem) likes = currentItem.likes;
+  return likes;
+};
+
+// mapping meal cards on the page
+const mapCard = (meals) => {
+  // empty the meals section first
+  mealsRow.innerHTML = '';
+
+  // loop through all given meals
+  meals.forEach(async (meal) => {
+    const likes = await showLikes(meal.idMeal);
+    // creating div element to hold one(1) meal info
+    const container = document.createElement('div');
+    container.classList.add('p-2', 'col-md-3', 'mb-2');
+    const mealCard = document.createElement('div');
+    mealCard.classList.add('meal-card', 'card', 'shadow-sm');
+    mealCard.innerHTML = `
+        <div class="meal-img"><img alt="${meal.strMeal}" src="${meal.strMealThumb}"></div>
+          <div class="d-flex flex-column justify-content-between px-3 py-2">
+            <div class="meal-desc mb-3">
+                <span class="meal-name ">${meal.strMeal}</span>
+            </div>
+            <div class="meal-action d-flex justify-content-between">
+                <div class="icon text-custom-color d-flex align-items-center">
+                 <div class="meal-likes" id="${meal.idMeal}">
+                </div>
+                    <span class="like-number rounded-circle ">${likes < 10 ? `0${likes}` : likes}</span>
+                    <button type="button" href="#" class=" btn btn-sm text-custom-color fa fa-thumbs-up like-icon" id="${meal.idMeal}"></button>
+                </div>
+                <div>
+                    <button type="button" class="btn btn-sm comment-btn fa fa-comment" id="${meal.idMeal}"></button>
+                    <button type="button" class="btn btn-sm reserv-btn fa fa-book" id="${meal.idMeal}"></button>
```

Comment:
- [ ] [OPTIONAL] Good job on implementing your project . It s awesome :100:  But I would like to recommend you remove it since is not part of the requirement for two project members and It is not working so useless :cry:  However it s very good to go beyond the requirement but you have to make sure that everything in your project works fine :smile: